lang/po/new/cs.po

```diff
@@ -733,7 +730,7 @@ msgstr "Merim referencni vysku kalibracniho bodu"
 # MSG_MESH_BED_LEVELING c=18
 #: messages.c:148
 msgid "Mesh Bed Leveling"
-msgstr ""
+msgstr "Mesh Bed Leveling"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.